### PR TITLE
Omit Obscure section

### PR DESCRIPTION
**Pull request by BrandonSharratt**
*Issue:*https://github.com/bcgov/entity/issues/22108

*Description of changes:*
Added omit obscure section

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-208-ifz3n6jy.web.app

File: btr-web/btr-main-app/components/bcros/RequestToOmitObscure.vue [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                />
+                <div class="inline-block mr-2">
+                  {{ $t('labels.requestOmit.omitInfoOpt.ALL') }}
+                </div>
+              </div>
+              <div
+                v-for="option, index in infoToOmitOptions"
+                :key="`omit${option.value}`"
+                class="inline-block mr-1 text-base"
+                :class="`w-[${option.width}]`"
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-main-app/components/bcros/RequestToOmitObscure.vue:134]: agree that was my last attempt to make them go into one line removing the hard set of width

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-208-ifz3n6jy.web.app

**Review by hfekete — APPROVED** [review]
OK, looks good now.

One thing though, when clicking omit all, you can still click birthday and all. If there were no requirements for those to be exclusive for each other, we can keep it like this for now and maybe raise it with the UX team.

**Pull request merged**